c10/macros/Macros.h

```diff
@@ -2,6 +2,8 @@
 #define C10_MACROS_MACROS_H_
 #include <cassert>
 
+#include <torch/standalone/macros/Macros.h>
```

Comment:
This is bad... we are creating cyclic dependencies between torch and c10. C10 is a tiny utility library and should be kept as such, even if this is a header only dependency, its breaks encapsulation quite badly here. We may have downstream projects wanting to use c10 without torch. Torchstandalone is welcome to bring in a copy of c10

Reply:
True, but it does violate encapsulation a bit, which is not great.